components/eam/src/physics/cam/cospsimulator_intr.F90

```diff
@@ -1382,6 +1391,7 @@ subroutine cospsimulator_intr_run(state,pbuf, cam_in,emis,coszrs,cld_swtau_in,sn
     ! MODIS outputs
     character(len=max_fieldname_len),dimension(nf_modis) :: &
          fname_modis=(/'CLTMODIS    ','CLWMODIS    ','CLIMODIS    ','CLHMODIS    ','CLMMODIS    ',&
+                       'CLTMODISIC  ','CLWMODISIC  ','CLIMODISIC ',&
```

Comment:
gnu on Anvil is reporting a build error (missing extra space?)
```
/gpfs/fs1/home/e3smtest/jenkins/workspace/ACME_Anvil_next/E3SM/components/eam/src/physics/cam/cospsimulator_intr.F90:1394:53:

                        'CLTMODISIC  ','CLWMODISIC  ','CLIMODISIC ',&
                                                     1
Error: Different CHARACTER lengths (12/11) in array constructor at (1)
```
for
- SMS.ne4_oQU240.F2010.anvil_gnu.eam-cosplite
- SMS_D_Ld1.ne30pg2_EC30to60E2r2.WCYCL1850.anvil_gnu.allactive-wcprod
- SMS_D_Ld1.ne30pg2_EC30to60E2r2.WCYCLSSP370.anvil_gnu.allactive-wcprodssp
- SMS_D_Ln5.ne4_oQU240.F2010.anvil_gnu.eam-cosplite_nhtfrq5